apps/core/components/rangeDatePicker/RangeDatePickerTypes.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import * as React from 'react';
+
+export type Props = {|
+  +isVisible: boolean,
+  +mode?: 'date' | 'time' | 'datetime',
```

Comment:
I don't think there is a plan to implement other modes, so we can probably remove this one.

Reply:
Removed